tensorflow/core/platform/s3/s3_file_system.cc

```diff
@@ -177,6 +177,7 @@ class S3RandomAccessFile : public RandomAccessFile {
 
   Status Read(uint64 offset, size_t n, StringPiece* result,
               char* scratch) const override {
+    VLOG(1) << "ReadFilefromS3 s3://" << bucket_ << "/" << object_ << " from " << offset << " for n:" << n;
```

Comment:
Instead of `1` we should use one of the constants to make this readable

Reply:
Will do, pending the discussion here: https://github.com/tensorflow/tensorflow/pull/36352#discussion_r382237079

Reply:
Let's keep them both as they are